### PR TITLE
Move tile-join to new jenkinsfile

### PR DESCRIPTION
**Pull request by mhines-usgs**

### CODE REVIEW

File: jenkins/tippecanoe_tile_join_Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      }
+      steps {
+        sh 'aws s3 sync tile_dir_simple5 s3://prod-owi-resources/resources/Application/wbeep/${TIER}/hru_shape/tile_dir_simple5'
+        sh 'tile-join --output-to-directory tiles -c hru_data.csv tile_dir_simple5 --force' 
+      }
+    }
+    stage('push to S3') {
+      agent any
+      steps { 
+        sh 'aws s3 sync tiles s3://wbeep-${TIER}-website/tiles'
```
Comment (mhines-usgs) [inline jenkins/tippecanoe_tile_join_Jenkinsfile]: eh wait, i guess this should be better to handle the metadata and contenttypes

File: jenkins/tippecanoe_create_tiles_Jenkinsfile [hunk 2]
```diff
@@ -23,13 +22,12 @@ pipeline {
...
       steps {
         sh 'tippecanoe -Z4 -z8 --no-tiny-polygon-reduction --detect-shared-borders --simplify-only-low-zooms --simplification=5 --force --output-to-directory tile_dir_simple5 hrus.geojson'
-        sh 'tile-join --output-to-directory tile_dir_simple5_data -c hru_data.csv tile_dir_simple5 --force' 
       }
     }
     stage('push to S3') {
       agent any
       steps { 
-        sh 'aws s3 sync tile_dir_simple5_data s3://prod-owi-resources/resources/Application/wbeep/${TIER}/hru_shape/tile_dir_simple5_data'
+        sh 'aws s3 sync tile_dir_simple5 s3://prod-owi-resources/resources/Application/wbeep/${TIER}/hru_shape/tile_dir_simple5'
```
Comment (mhines-usgs) [inline jenkins/tippecanoe_create_tiles_Jenkinsfile]: figured it makes sense to leave these "non-data" tiles/naked tiles in this bucket? then grab them during the join and then push out to the specified tier?

**Comment by mhines-usgs** [comment]
also created a copy of your tippecanoe_build_tileset job and made a tippecanoe_tile_join job if you don't mind checking that out!

File: jenkins/tippecanoe_create_tiles_Jenkinsfile (hunk 2 above)
Comment (wdwatkins) [reply jenkins/tippecanoe_create_tiles_Jenkinsfile]: agree

File: jenkins/tippecanoe_tile_join_Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    }
+    stage('create tileset') {
+      agent {
+        docker {
+          image 'code.chs.usgs.gov:5001/wma/iidd/wbeep-data-processing:tippecanoe-latest'
+          alwaysPull true
+        } 
+      }
+      steps {
+        sh 'aws s3 sync tile_dir_simple5 s3://prod-owi-resources/resources/Application/wbeep/${TIER}/hru_shape/tile_dir_simple5'
```
Comment (wdwatkins) [inline jenkins/tippecanoe_tile_join_Jenkinsfile]: Could do this up with the other S3 pull, not sure it really matters though?

File: jenkins/tippecanoe_tile_join_Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+            label 'node:slave'
+        }
+    }
+  stages {
+    stage('Checkout repo and pull from S3') {
+      agent any 
+      steps {
+        sh 'wget -O DOIRootCA2.cer http://sslhelp.doi.net/docs/DOIRootCA2.cer'
+        git "https://github.com/usgs-makerspace/wbeep-processing"
+        sh 'aws s3 sync s3://prod-owi-resources/resources/Application/wbeep/${TIER}/hru_data . --exclude "*" --include "hru_data.csv"'
```
Comment (wdwatkins) [inline jenkins/tippecanoe_tile_join_Jenkinsfile]: What file name should we use for the .csv?  I called it `model_output_categorized.csv`

**Comment by wdwatkins** [comment]
Jenkins job looks good in the UI.  We may want to set these up with the include/exclude regions and SCM polling, so they are only triggered by changes to relevant files.  Not a must-do right now though

File: jenkins/tippecanoe_tile_join_Jenkinsfile (hunk 4 above)
Comment (mhines-usgs) [reply jenkins/tippecanoe_tile_join_Jenkinsfile]: ok, i'll use that then. and it looks like you just have it at the root of the tier folder?

**Pull request merged**